### PR TITLE
More projects

### PR DESCRIPTION
**Pull request by derekhiggins**

### CODE REVIEW

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/218/

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/223/

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/224/

File: run_ci.sh [hunk 1]
```diff
@@ -38,6 +38,9 @@ sudo sed -i -e '/keepcache=0/d' /etc/yum.conf
 sudo mkdir -p /opt/data/yumcache
 sudo mount -o bind /opt/data/yumcache /var/cache/yum
 
+# TEST no merge
+git clone https://github.com/knowncitizen/facet -b openshift-metalkube-migration
```
Comment (jtomasek) [inline run_ci.sh]: this can be updated as https://github.com/openshift-metalkube/facet/pull/49 has been merged

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/231/

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/286/

**Pull request merged**